OHHTTPStubs/UnitTests/Test Suites/NSURLSessionTests.m

```diff
@@ -116,13 +119,15 @@ - (void)_test_redirect_NSURLSession:(NSURLSession*)session
             return [[[request URL] path] isEqualToString:@""];
         } withStubResponse:^OHHTTPStubsResponse *(NSURLRequest *request) {
             NSDictionary *headers = @{ @"Location": @"foo://unknownhost:666/elsewhere" };
-            return [[OHHTTPStubsResponse responseWithData:[[NSData alloc] init] statusCode:301 headers:headers]
+            NSLog(@"Stop here");
```

Comment:
Don't forget to remote that NSLog once you've done with experimenting on that test before we merge :wink:

Reply:
It's already out in my code.  🥇